src/test/kotlin/org/elm/ide/inspections/ElmIncompletePatternInspectionTest.kt

```diff
@@ -90,6 +90,137 @@ foo it =
             --EOL
 """)
 
+//    fun `test no existing branch in parenthesis`() = checkFixByText("Add missing case branches", """
```

Comment:
On `master`, this test fails with this diff:
![image](https://user-images.githubusercontent.com/6695040/103339312-560f1e00-4a81-11eb-9473-5a10408d79ed.png)

Reply:
On my branch, this test fails with this diff:
![image](https://user-images.githubusercontent.com/6695040/103339683-50660800-4a82-11eb-88d5-12e4f3301658.png)

Reply:
Arguably, the new code produces worse output here. Using this inspection on a text ala `(case it of)` may also seem somewhat contrived, but I often find that I need a `case` inside parentheses, so it's very natural for me to start with the parentheses, and fill in the code from there. For me, this case is really the one I would like to fix.